hazelcast/src/main/java/com/hazelcast/client/impl/clientside/ClientExceptionFactory.java

```diff
@@ -329,7 +329,11 @@ private Throwable createException(Iterator<ErrorHolder> iterator) {
         if (exceptionFactory == null) {
             throwable = new UndefinedErrorCodeException(errorHolder.getMessage(), errorHolder.getClassName());
```

Comment:
An off-topic proposal. This branch could be further improved to handle causes for unknown exceptions. See https://github.com/hazelcast/hazelcast-nodejs-client/pull/576#discussion_r477257939 for more details.